.github/workflows/reusable_provenance.yaml

```diff
@@ -124,16 +124,13 @@ jobs:
         run: |
           echo "provenance_digest=$(ent put ${{ needs.get_inputs.outputs.provenance-name }}.sigstore)" >> $GITHUB_OUTPUT
 
-      # Add the provenance digest to the ./comment file.
+      # Add the provenance digest to the ./comment file as a TOML formatted doc.
```

Comment:
Perhaps we should stop posting them as comments at some point, if we expect them to be machine readable. What is the longer term plan?

Reply:
What do you suggest as the alternative?
I generally prefer as much automation as possible. In the current form, they are easy to access for the human user too.

Reply:
I remember we discussed some alternative index hosted somewhere else (like we used to have in git), or something based on Ent. I think the current solution is ok as a temporary stopgap, but I don't imagine we will want to keep posting TOML comments. Also I think this would not work if we move the source of truth to git-on-borg anyways.

Reply:
The provenance branch in Git was designed for a different use case, which we thought was covered by Rekor's search API. The importer use case, which is the reason for this change, is slightly different. I think it makes sense at this stage to revisit all use cases and see if we can design an index that addresses all use cases.